src/components/swap/PendingModalContent.test.tsx

```diff
@@ -0,0 +1,138 @@
+import { render, screen } from 'test-utils/render'
+
+import { PendingModalContent } from './PendingModalContent'
+
+describe('PendingModalContent', () => {
+  it('should render null with invalid content', () => {
+    const result = render(
+      <PendingModalContent steps={[]} activeStepIndex={0} confirmed={false} transactionSuccess={false} />
+    )
+    expect(result.container).toBeEmptyDOMElement()
+  })
+
+  it('should render the only step', () => {
```

Comment:
this test name is slightly confusing. it's basically saying if there's only one step we render it?